qcodes/instrument_drivers/Keysight/private/Keysight_344xxA_submodules.py

```diff
@@ -907,6 +908,68 @@ def autorange_once(self) -> None:
         self.write('SENSe:VOLTage:DC:RANGe:AUTO ONCE')
         self.range.get()
 
+    def increase_range(
+        self,
+        range_value: Optional[float] = None,
+        increase_by: int = 1
+    ) -> None:
+        """
+        Increases the voltage range by a certain amount with default of 1.
+        If limit is reached, the max range is used.
+
+        Args:
+            range_value: The desired voltage range needed.  Expressed by power
+                of 10^x range from -3 to 10
+            increase_by: How much to increase range by, default behavior
+                 is by a step of one.
+
+        """
+        if increase_by < 1:
+            raise ValueError("The steps must be increasing in value")
+
+        current_range = 1.0 #default to 10 ** 0
```

Comment:
This line has no effect right as we either use the if or else branch?

Reply:
Should be good to go now.